cid/helpers/iam.py

```diff
@@ -178,6 +178,27 @@ def ensure_role_does_not_exist(self, role_name):
         except self.client.exceptions.DeleteConflictException:
             logger.debug(f'Role  {role_name!r} is still used. Skipping')
 
+    def create_role(self, role_name: str, assume_role_policy_document: str) -> str:
+        """Create IAM role with a specified name and role policy document for assuming the role"""
+        response = self.client.create_role(RoleName=role_name, AssumeRolePolicyDocument=assume_role_policy_document, Tags=self.default_tag_list)
```

Comment:
This can fail with many reasons: 
- role can exist and permissions are the main ones

Reply:
Good point. I'm letting that exception bubble up and I'm handling it in `_create_glue_crawler_role` in the `InitCurCommand`, because if we want to use this method anywhere else, I would probably handle the exception differently there